packages/office-ui-fabric-react/src/utilities/DraggableZone/DraggableZone.types.ts

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { IRefObject } from '../../Utilities';
+
+export interface IDragData {
+  position: ICoordinates;
+  lastPosition?: ICoordinates;
+  delta: ICoordinates;
+}
+
+export interface ICoordinates {
+  x: number;
+  y: number;
+}
+
+export interface IDraggableZone {}
+
+export interface IDraggableZoneProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Optional callback to access the IDraggableZone interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: IRefObject<IDraggableZone>;
+
+  /**
+   * Specifies a selector to be used as the handle that initiates drag
+   */
+  handleSelector?: string;
+
+  /**
+   * Specifies a selector to be used to prevent drag initialization
+   */
+  preventDragSelector?: string;
```

Comment:
I don't understand the use case of this. Can you update the comment a bit? When should I specify this?

Reply:
Thanks, that makes sense.